Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN apk upgrade --no-cache && \
 
 COPY bin/run.sh /run.sh
 COPY conf.d /etc/nginx/conf.d
+COPY tests/nginx.conf /nginx.conf
```

Comment:
Does this mean that test config will be included in a final image? If that's the case, you want a separate Dockerfile for testing or a way to enable copying the test config only at test time.

Reply:
It does, I didn't think about that. I will change this.

Reply:
You could consider using a [multi-stage docker build](https://docs.docker.com/engine/userguide/eng-image/multistage-build/)